openeo/testing/results.py

```diff
@@ -250,7 +405,10 @@ def assert_xarray_allclose(
     if isinstance(actual, xarray.Dataset) and isinstance(expected, xarray.Dataset):
         assert_xarray_dataset_allclose(actual, expected, rtol=rtol, atol=atol)
     elif isinstance(actual, xarray.DataArray) and isinstance(expected, xarray.DataArray):
-        assert_xarray_dataarray_allclose(actual, expected, rtol=rtol, atol=atol)
+        if (["x", "y", "band"]).elements_in(expected.dims):
```

Comment:
This doesn't look like valid python

you probably mean something like 
```suggestion
        if {"x", "y", "band"}.issubset(expected.dims):
```

I'm also not sure why "bands" is included in this check